docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,347 @@
+    you can optionally replace the target `pvc` sections in your `vmTemplate` resources with a `storage` section once
+    you have defined a storage profile.
+
+    Using storage instead of PVC will automatically discover default values for `accessMode`, `volumeMode` and
+    optionally even the size from the source PVC and its associated storage profile.
+
+    A benefit of `storage` over `pvc` is that the CDI will automatically calculate the size overhead for `Filesystem`
+    mode storage classes, allowing the VM template to use the original disk size instead of one adjusted for the
+    filesystem overhead.
+
+    If you want to use this template, replace the contents of the second manifest with the following snippet.
+
+    <br />
+
+    ```yaml {14}
+    apiVersion: spectrocloud.com/v1
+    kind: VmTemplate
+    metadata:
+      name: ubuntu-2204
+    spec:
+      dataVolumeTemplates:
+        - metadata:
+            name: ubuntu-2204
+          spec:
+            source:
+              pvc:
+                name: template-ubuntu-2204
+                namespace: vmo-golden-images
+            storage:
+              resources:
+                requests:
+                  storage: 50Gi
+      template:
+        metadata:
+          annotations:
+            descheduler.alpha.kubernetes.io/evict: "true"
+        spec:
+          domain:
+            cpu:
+              cores: 2
+              sockets: 1
+              threads: 1
+            devices:
+              disks:
+                - disk:
+                    bus: virtio
+                  name: datavolume-os
+                - disk:
+                    bus: virtio
+                  name: cloudinitdisk
+              interfaces:
+                - masquerade: {}
+                  name: default
+                  model: virtio
+            machine:
+              type: q35
+            resources:
+              limits:
+                memory: 2Gi
+              requests:
+                memory: 2Gi
+          networks:
+            - name: default
+              pod: {}
+          volumes:
+            - dataVolume:
+                name: ubuntu-2204
+              name: datavolume-os
+            - cloudInitNoCloud:
+                userData: |
+                  #cloud-config
+                  ssh_pwauth: True
+                  chpasswd: { expire: False }
+                  password: spectro
+                  disable_root: false
+                  runcmd:
+                    - apt-get update
+                    - apt-get install -y qemu-guest-agent
+                    - systemctl start qemu-guest-agent
+              name: cloudinitdisk
+    ```
+
+6.  Click on **Confirm & Create**.
+
+7.  Complete cluster profile configuration and click on **Finish Configuration**.
+
+## Validate
+
+1.  Navigate to the left **Main Menu** and select **Clusters**.
+
+2.  Select your cluster and view its **Overview** tab. Select the **Profile** tab.
```

Comment:
```suggestion
2.  Select your cluster and click the **Profile** tab.
```

Reply:
I'd remove `and view its **Overview** tab`, since it doesn't seem to relate to the actual validation step